ghost/core/core/frontend/helpers/title.js

```diff
@@ -1,7 +1,7 @@
 // # Title Helper
 // Usage: `{{title}}`
 //
-// Overrides the standard behaviour of `{[title}}` to ensure the content is correctly escaped
+// Overrides the standard beharior of `{[title}}` to ensure the content is correctly escaped
```

Comment:
Seems a find and replace went wrong here?

Reply:
Yes it did :grimacing: https://github.com/TryGhost/Ghost/commit/fa8d94fce200c915f95dfc9c9a14b967ff7aa50c